httlemon/collections.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from tinydb import TinyDB
+from tinydb.storages import JSONStorage
+
+
+DB_NAME = 'httlemon/httlemon_db.json'
```

Comment:
this database should be stored in a more usual place like `.config/httlemon.json` or simply `.httlemon/db.json` in `$HOME` (note it should be a hidden folder imo, and if you're already under a `httlemon` dir no need to add it as a prefix to the file).